tasks.js

```diff
@@ -1,5 +1,6 @@
 import {createModal} from './modal.js';
-import {appendTaskInColumn, counterForDeleted} from './index.js';
+import {appendTaskInColumn, counterForDeleted, getSearchParams, setSearchParams} from './index.js';
```

Comment:
for what do you import setSearchParams here?

Reply:
for appendDeleteTaskInColumn in tasks.js